example/nce-loss/README.md

```diff
@@ -54,28 +63,26 @@ The motivation is to design a more robust and scalable word vector system, by re
 
 ### Basics
 
-<img src="https://github.com/zihaolucky/mxnet/blob/example/word2vec-nce-loss-with-subword-representations/example/nce-loss-subword-repr/slide1.png" width="700">
+<img src="images/slide1.png" width="700">
 
-<img src="https://github.com/zihaolucky/mxnet/blob/example/word2vec-nce-loss-with-subword-representations/example/nce-loss-subword-repr/slide2.png" width="700">
+<img src="images/slide2.png" width="700">
```

Comment:
Basically we don't put data like image into main source code, as that makes our code base too large.

Reply:
Sure. But linking to images from an unknown repo can break the README in the future. Also I followed this example where the image is part of the repo: https://github.com/apache/incubator-mxnet/tree/master/example/captcha
Is linking to the image in a personal repo preferred?!

Reply:
Alternatively I can just remove the images and leave it to user to look up the paper in the link.